### PR TITLE
[10.0][MIGRATION] Migrated module 'date_range' to V10

### PR DESCRIPTION
**Pull request by espo-tony**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: date_range/README.rst [hunk 1]
```diff
@@ -91,6 +91,8 @@ Contributors
 ------------
 
 * Laurent Mignon <laurent.mignon@acsone.eu>
+* Andrea Stirpe <a.stirpe@onestein.nl>
```
Comment (lmignon) [inline date_range/README.rst]: IMO this kind of changes doesn't justify 2 new contributors. But thank you for the migration.

**Review by lmignon — CHANGES_REQUESTED** [review]

**Review by Garamotte — APPROVED** [review]
Same remark as @lmignon 
Otherwise :+1:

**Comment by espo-tony** [comment]
@lmignon , @sylvain-garancher 
Sorry for updating the contributor list without a proper reason; I just corrected my mistake.

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Review by lmignon — APPROVED** [review]

**Comment by lmignon** [comment]
@espo-tony You're welcome :smirk: Your name is in the commit history and you have done a valuable work! I just try to avoid that the Contributor's list become a list of all commiters. Nevertheless if you enhance the module (technically / functionally) I'll be happy to see your name in this list.

**Pull request merged**